liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -30,6 +30,20 @@ using namespace solidity::langutil;
 using namespace solidity::util;
 using namespace solidity::util::formatting;
 
+namespace
+{
+
+std::string replaceNonTabs(std::string const& _utf8Input, size_t _pos, size_t _len, char _filler)
+{
+	std::string output;
+	for (size_t i = _pos; i < _pos + _len; ++i)
+		if (char const c = _utf8Input.at(i); (c & 0xc0) != 0x80)
```

Comment:
`(c & 0xc0) != 0x80)` What does this do?

Reply:
I did figure it out, thanks to SO. I wonder if there is an inbuilt function that can help us here.

Reply:
The same comparison is also used in UTF8.cpp. Maybe it can be merged, though not sure how worthwhile it would be.

Reply:
> The same comparison is also used in UTF8.cpp

Sure, I borrowed it from there. At first I even added the helper function to UTF8.cpp, but then decided that it is rather for local use.

Reply:
Is that function in UTF even still used? I get the impression your code replaced the usage of it. We might as well remove it then

Reply:
No, but the function was pretty, I could not remove it in cold blood. OK, 5c07654.